ui/assets/excluded_files.tpl.html

```diff
@@ -9,8 +9,11 @@
     <body>
         <div id="root"></div>
 
-        <script src="js/react-{{.ReactVersion}}.min.js"></script>
+        <script src="js/react-with-addons-{{.ReactVersion}}.min.js"></script>
+        <script src="js/react-dom-{{.ReactVersion}}.min.js"></script>
         <script src="js/jquery-{{.jQueryVersion}}.min.js"></script>
+        <script src="js/babel-core.5.8.23.browser.min.js"></script>
+        <script src="js/common.js"></script>
```

Comment:
Previously, in production builds, we were fetching 2 external scripts and then had the whole of the main script inline. We're now fetching 5 isolated scripts. I'd rather bundle some more scripts together than to continue accumulating more HTTP requests. Also, are all of these files needed in production builds? I had assumed that babel, for instance, would only be needed in `--dev` mode.